site/js/app.js

```diff
@@ -1,16 +1,32 @@
 
 import Flooper from '../../dist/Flooper.js';
 import {consoleColophon} from './consoleColophon';
+import {toggleAnimation} from './utils';
 
 consoleColophon();
-var i = new Flooper();
-i.init();
+var demoFlooper = new Flooper();
+demoFlooper.init();
+
+var paused = false;
 
 document.querySelector(".js-flooper-pause").onclick = () => {
-  i.pause();
+  demoFlooper.pause();
+
+  if(!paused) toggleAnimation(document.querySelector(".js-flooper"), "paused");
```

Comment:
We should provide a method like `getState()` to retrive expose animation state variables

```javascript
//lib
getState(){
  return {
     paused: this.paused,
     slowmotion: this.slowmotion
   }
}

// In app.js
var demoFlooper = new Flooper();
demoFlooper.init();

var paused = demoFlooper.getState().paused;
```